src/__tests__/use-stepper.test.tsx

```diff
@@ -67,57 +67,70 @@ describe('useStepper', () => {
   it('returns the correct properties', () => {
     const { result } = renderHook(() => useStepper());
     expect(result.current).toMatchInlineSnapshot(`
-      {
-        "decrement": [Function],
-        "getDecrementProps": [Function],
-        "getFormProps": [Function],
-        "getIncrementProps": [Function],
-        "getInputProps": [Function],
-        "increment": [Function],
-        "setValue": [Function],
-        "value": "0",
-      }
+    {
+      "decrement": [Function],
+      "getDecrementProps": [Function],
+      "getFormProps": [Function],
+      "getIncrementProps": [Function],
+      "getInputProps": [Function],
+      "increment": [Function],
+      "setValue": [Function],
+      "value": "0",
+    }
```

Comment:
All these inline snapshots need indented two more spaces, but now I'm just being nitpicky. 😅 I can fix it later, nbd.